src/css/Autocomplete.css

```diff
@@ -19,6 +19,6 @@
   background: white;
   box-shadow: 0 2px 12px rgba(0, 0, 0, 0.1);
   opacity: 1 !important;
-  position: fixed;
```

Comment:
What does `top: auto;` control? Noticed a `position `property was removed here, which is usually the pre-requisite for using `top: insert number here;`

Reply:
@theilman7 top:auto ensure the autocomplete results are tucked just below the autocomplete input, the current library are using, uses javascript to figure out the top value, but it is wrong. This fixes it.